src/thunderbots/backend_output/grsim/grsim_backend.h

```diff
@@ -1,13 +1,62 @@
 #ifndef PROJECT_GRSIM_BACKEND_H
 #define PROJECT_GRSIM_BACKEND_H
+#include <boost/asio.hpp>
+#include <string>
 #include "backend_output/backend.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
 
 class GrSimBackend : public Backend
 {
    public:
-    explicit GrSimBackend();
+    /**
+     * Creates a new grSim backend
+     *
+     * @param network_address The IP address to publish grSim commands to
+     * @param port The port to publish commands to
+     */
+    explicit GrSimBackend(std::string network_address, unsigned short port);
+    ~GrSimBackend();
+
+    /**
+     * Sends the given primitives to grSim
+     * @param primitives the list of primitives to send
+     */
     void sendPrimitives(const std::vector<Primitive> &primitives) override;
-};
 
+   private:
+    /**
+     * Sets the velocities of a robot in grSim. The velocities are in the Robot's local
+     * coordinates.
+     *
+     * @param robot_id The id of the robot to send the command to
+     * @param team_colour_yellow Specifies if the robot to send the command to is on the
+     * yellow team
+     * @param velocity The velocity to set for the robot, in robot coordinates. X
+     * corresponds to "forward" from the robot's perspective, and Y corresponds to the
+     * left of the robot. Values are in metres per second.
+     *
+     *            X (forward)
+     *            ^
+     *            |
+     *            |
+     *   Y <--- Robot
+     *
+     * @param angular_velocity The angular velocity to set for the robot, in Radians per
+     * second.
+     */
+    void setRobotVelocities(
+        unsigned int robot_id, bool team_colour_yellow, Point velocity,
```

Comment:
Is this what clang-format changed this to? Looks a bit funny...

Reply:
Yes. Line length is currently 90. Thoughts on increasing it? I believe CLion's default marker is 120 if that gives some context.

Reply:
I am strong against increasing the line length further for several reasons
- makes it harder to work with on smaller screens
- using a shorter line length forces people to break things up and makes it more readable

I agree this isn't ideal here, but there are several options in clang-format we can change to make this nicer: `AlignAfterOpenBracket`, `AllowAllParametersOfDeclarationOnNextLine`, `AllowShortFunctionsOnASingleLine`,  `BinPackArguments`, `BinPackParameters`, `IndentWrappedFunctionNames` are some of the ones that can tweak how a function definition is formatted (all are described in more detail here: https://clang.llvm.org/docs/ClangFormatStyleOptions.html)

Reply:
We can discuss this in today's meeting. Depending on the rest of the review we can either re-format in this PR or the next one.

Reply:
Yeah, what I kind of like instead is     
```
void setRobotVelocities(unsigned int robot_id, bool team_colour_yellow, 
                                      Point velocity, ...
```

Reply:
@henrymliu - I'm cool with that, didn't sound like anyone had a strong opinion on this, so feel free to change and reformat repo (@MathewMacDougall ?)